amplify/backend/function/questionslist/src/index.js

```diff
@@ -0,0 +1,14 @@
+const { parseQuestions, batchWriteQuestions } = require('./questions');
+
+const MAX_ITEMS_PER_DB_WRITE = 25;
+
+exports.handler = async (event) => {
+  const questions = await parseQuestions();
+  for (let i = 0; i < questions.length / MAX_ITEMS_PER_DB_WRITE; i += 1) {
```

Comment:
Some documentation (i.e. comments) here as to why you divide questions.length / MAX_ITEMS_PER_DB_WRITE would be nice (I'm guessing this is because you know the number of questions is constant, so you'll always get a quotient > 0)